corehq/apps/case_importer/templates/case_importer/import_cases.html

```diff
@@ -2,6 +2,53 @@
 {% load report_tags %}
 {% load i18n %}
 
+{% block js-inline %}
+<script id="caseUploadsResource" type="text/plain">{% url 'case_importer_uploads' domain %}</script>
```

Comment:
Yeah good question. If I put `"{% url ''case_importer_uploads' domain %}"` inline, then it's escaping for an html context but putting in a string-in-javascript context, so there are a number of things that can go wrong, even though it's rare. Now that I think of it though, I believe `{% url ''case_importer_uploads' domain as url %}{{ url|JSON }}` is the simplest way to do this correctly, and I think that's a better convention, so I'll switch to that. Thanks for asking